File: HandwritenRecognition.Cross/DataTransferObjects/OcrResultDto.cs
namespace HandwritenRecognition.Cross.DataTransferObjects;

public class OcrResultDto
{
    public List<OcrLineDto> Lines { get; set; } = new();
    public Guid DocumentId { get; set; }
    public List<ImageInfoDto> ImageInfo { get; set; } = [];
    public List<QualityInfoDto> QualityInfo { get; set; } = [];
    public List<ProcessDto> ProcessData { get; set; } = [];
}

File: handwritten-recognition-client/src/model/dto/OcrResultDto.ts
import type { ImageInfoDto } from "./ImageInfoDto";
import type { OcrLineDto } from "./OcrLineDto"
import type { ProcessDto } from "./processDto";

export type OcrResultDto={
    lines:Array<OcrLineDto>;
    documentId:string,
    imageInfo:ImageInfoDto,
    processInfo:ProcessDto
}